examples/new-client.rs

```diff
@@ -0,0 +1,15 @@
+use nats::*;
+use std::io;
+
+fn main() -> io::Result<()> {
+    // Useful commands for testing:
+    // nats-sub -s nats://demo.nats.io:4222 hello
```

Comment:
don't need nats:// or the 4222 which is default.

`nats-sub -s demo.nats.io hello`